packages/playground/src/views/solutions_view.vue

```diff
@@ -30,40 +30,52 @@ export default {
           "Nostr is a simple, open protocol that enables global, decentralized, and censorship-resistant social media.",
         icon: "nostr.png",
         route: DashboardRoutes.Applications.Nostr,
+        isNew: false,
+        createdAt: 1725283777169, // Update creation time in milliseconds ex: 1723468996310 is 'Mon Aug 12 2024 16:23:16'
```

Comment:
Instead of hard coding the timestamps, what if we created a function to create timestamps and call it here?

Reply:
I think it can't be dynamic as it will be called each time the component is mounted so it will get the date of the current day each time